### PR TITLE
overhauled the regions to fill in the continents and regions

### LINKED ISSUE #14: regions.js is missing North America
You've got many countries listed elsewhere, but no Canada or US. 
http://en.wikipedia.org/wiki/List_of_sovereign_states_and_dependent_territories_in_North_America

### PR DESCRIPTION
**Pull request by iancrowther**
resolves #14

### CODE REVIEW

File: data/regions.js [hunk 1]
```diff
@@ -237,106 +190,261 @@ regions.westernAfrica = {
...
+  countries: _.flatten([
+    regions.northernEurope.countries,
+    regions.southernEurope.countries,
+    regions.easternEurope.countries,
+    regions.westernEurope.countries,
+  ])
+  .sort()
+}
+ 
+regions.australia = {
```
Comment (evdb) [inline data/regions.js]: Should this be `regions.australasia` as suggested by http://en.wikipedia.org/wiki/Australasia? In which case New Guinea should be included somehow...

File: data/regions.js [hunk 2]
```diff
@@ -237,106 +190,261 @@ regions.westernAfrica = {
...
     'TO', // Tonga
     'TV', // Tuvalu
-    'VU', // Vanuatu
+    'WF', // Wallis and Futuna
   ]
 };
+
+regions.oceania = {
+  countries: _.flatten([
+    regions.australia.countries,
```
Comment (evdb) [inline data/regions.js]: would need changing as per note above

File: data/languages.csv [hunk 3]
```diff
@@ -337,16 +337,17 @@ Mirandese,,mwl,
...
 Mong,,hmn,
 Mongo,,lol,
 Mongolian,mn,mon,
+Montenegrin,,mot,
 Mossi,,mos,
 Multiple languages,,mul,
 Munda languages,,mun,
 N'Ko,,nqo,
 Nahuatl languages,,nah,
+name,alpha2,alpha3,bibliographic
```
Comment (evdb) [inline data/languages.csv]: Accidentally added header line :)

**Comment by evdb** [comment]
This all looks good - bar a few comments that I've made inline. Main concern is the `regions.australia` which I think should be `regions.australasia` but might well be wrong.

Thanks for the changes and additions - I know it can be a very time consuming thing to do :)

**Comment by iancrowther** [comment]
ok - I had it as australasia originally, having been deep in wikipedia got the perception that oceania was a better fit, however its 50/50.  

Do you still want the changes?

and yes, its much bigger than I first thought :-)

have tests now btw to sync the countries json & regions.js

**Comment by evdb** [comment]
The tests would be great - always keen to have tests :)

Not sure which of Australasia or Oceania are better. It seems that the exact definitions are rather vague... I think change it to Australasia as that seems most precise.

**Comment by iancrowther** [comment]
np - will do

**Comment by iancrowther** [comment]
Hey @evdb 

I took a look at the [United Nations](http://unstats.un.org/unsd/methods/m49/m49regin.htm) decomposition of regions and it turns out they refer to the region as oceania.

As much as I love Wikipedia, I think the UN is most internationally recognised.

Ian

**Comment by iancrowther** [comment]
closing to push updates, clensing and tests

**Pull request closed without merging**